static/reverse-context.js

```diff
@@ -20,6 +20,15 @@ window.__karma__ = (function(hasParent) {
 
 	function callParentKarmaMethod(methodName, args) {
 		args.unshift('iframe-test-results', methodName);
+		for (var i = 0, l = args.length; i < l; ++i) {
```

Comment:
You could start the loop at `i = 2` since you know items 0 and 1 are `'iframe-test-results'` and the methodName, respectively.